Add support for libvirt

jbouter opened the pull request:
- Update README to facilitate libvirt usage
- Add Vagrantfile-libvirt

This new vagrantfile differs from the original:
- Use centos/7 instead of bento/centos-7

jbouter committed:
Add support for libvirt

- Update README to facilitate libvirt usage
- Add Vagrantfile-libvirt

This new vagrantfile differs from the original:
- Use centos/7 instead of bento/centos-7

Signed-off-by: Jeffrey Bouter <jb@warpnet.nl>

haijeploeg commented on README.md:
````diff
@@ -9,6 +9,25 @@ This setup has been tested on Ubuntu 18.04 LTS and MacOS Catalina. The following
+Install Vagrant:
+
+```bash
+apt install -y vagrant
````
Don't install vagrant using apt. Download it from the vagrant website instead. The one in the apt repository is very old.

jbouter replied:
Very old indeed, but the only one that works out of the box with vagrant-libvirt.

jbouter committed:
Update vagrant for libvirt

- Fix Vagrantfile to allow repo-based libvirt
- Update README with clarification on installation
- Update ansible playbooks to support command line installations

Signed-off-by: Jeffrey Bouter <jb@warpnet.nl>
